net.veloren.veloren.yaml

```diff
@@ -55,7 +55,8 @@ modules:
       #- sed -i 's/opt-level = 2/opt-level = 0/g' Cargo.toml
 
       - cargo --offline fetch --manifest-path Cargo.toml
-      - cargo --offline build --release
+      - cargo --offline build -p veloren-voxygen --release --no-default-features --features="default-publish"
+      - cargo --offline build -p veloren-server-cli --release
```

Comment:
Not a requirement, but can we build both `veloren-voxygen` and `veloren-server-cli` in one run? Looks like the second `cargo build` almost doubles the build time.

Reply:
Done - we removed the requirement to build both separately upstream and I've updated this PR accordingly

Reply:
Thanks, I see it now builds in 30 minutes again.